pages/collaboration/team.mdx

```diff
@@ -0,0 +1,35 @@
+# Team Collaboration
+
+After having signed up to ModelZ, you’ve automatically received a personal account that you can use to run and deploy personal models. To deploy collaborate with others, however, you can create or join additional teams.
+
+## Create a Team
+
+You can create a team, and fill in your team member's e-mail address and role. 
+- If the email of team member is already signed up in ModelZ, the member will receive a team invitation by e-mail. After accepting the invitation, he can see the team in his team list.
```

Comment:
Maybe `the email of team member` to be `invited user of team`?

Reply:
lgtm